server/spectatord.cc

```diff
@@ -346,11 +346,15 @@ void Server::Start() {
     logger->info("statsd support is not enabled");
   }
 
-  prepare_socket_path(socket_path_);
-  LocalServer dgram_local{io_context, socket_path_, parser};
-  dgram_local.Start();
-  logger->info("Starting local server (dgram) on socket {}", socket_path_);
-  io_context.run();
+  if (socket_path_ != "**disabled**") {
```

Comment:
Probably better to use the boolean flag instead of comparing it against the magic value. 

```
if (absl::GetFlag(FLAGS_enable_socket)) {
```

Reply:
If we do that, should we do the same for the statsd_enable flag, which uses -1 as a sentinel (versus "**disabled**")?